boa/contracts/verify.py

```diff
@@ -0,0 +1,74 @@
+import json
+from datetime import datetime, timedelta
+
+import requests
+from attr import dataclass
+
+from boa.util.abi import Address
+
+
+@dataclass
+class Blockscout:
+    """
+    Allows users to verify contracts on Blockscout.
+    This is independent of Vyper contracts, and can be used to verify any smart contract.
+    """
+
+    api_key: str
+    api_url: str = "https://eth.blockscout.com"
+
+    def verify(
+        self,
+        address: Address,
+        contract_name: str,
+        standard_json: dict,
+        evm_version: str,
+        license=None,
+    ) -> None:
+        """
+        Verifies the Vyper contract on Blockscout.
+        :param address: The address of the contract.
+        :param contract_name: The name of the contract.
+        :param standard_json: The standard JSON output of the Vyper compiler.
+        :param evm_version: The EVM version to use for verification.
+        :param license: The license to use for the contract. Defaults to "none".
+        """
+        response = requests.post(
+            url=f"{self.api_url}/api/v2/smart-contracts/{address.lower()}/"
+            f"verification/via/vyper-standard-input?apikey={self.api_key}",
+            data={
+                "compiler_version": standard_json["compiler_version"],
+                "license_type": license or "none",
```

Comment:
what if the user passes `license = ""`? i think it's better to have an explicit line `if license is None: license = "none"`

Reply:
`license=""` would imply `"none"` rather than just raise an error, but yeah, let users make their own mistakes